spec/services/list_volunteer_opportunities_spec.rb

```diff
@@ -0,0 +1,43 @@
+require './app/services/list_volunteer_opportunities'
+
+describe ListVolunteerOpportunities do
+
+  let(:listener) { double :listener }
+  let(:organisations) { double :organisations }
+  let(:feature) { double :feature}
+  let(:http_party) { double :http_party}
+  let(:harrow_markers) { '[]'}
+
+  subject(:list_volunteer_opportunities) { ListVolunteerOpportunities.with(listener, organisations, feature, http_party) }
```

Comment:
Metrics/LineLength: Line is too long. [122/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)